src/components/tier.jsx

```diff
@@ -17,15 +17,13 @@ class Tier extends React.Component {
       }
     }
     this.getTierData = this.getTierData.bind(this);
-  }
 
-  componentDidMount() {
     this.setState({
       tierId: this.props.tierId,
       tierCost: this.props.tierCost,
       tierType: this.props.tierType,
     });
-    this.getTierData();
+    // this.getTierData();
```

Comment:
Checking in... is this cruft or just commented out while you work on things?

Reply:
Just commented out since that API call isn't working right now. Saves me from having to see other errors popping up unrelated to what I was working on :-)